MetricsUploader/include/MetricsUploader/MetricsUploader.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+#define ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+
+#ifdef _WIN32
+#include <windows.h>
+#endif  // WIN32
+
+#include <absl/synchronization/mutex.h>
+
+#include <chrono>
+#include <cstdint>
+#include <memory>
+#include <string>
+#include <string_view>
+
+#include "MetricsUploader/Result.h"
+#include "OrbitBase/Result.h"
+#include "orbit_log_event.pb.h"
+
+namespace orbit_metrics_uploader {
+
+const std::string kMetricsUploaderClientDllName = "metrics_uploader_client";
+
+// This class is used for sending log events from Orbit. It works only on Windows and only if
+// metrics_uploader_client.dll is available. The types of logs that could be send are defined in
+// orbit_client_protos::OrbitLogEvent::LogEventType.
+//
+// Only one instance of the class is allowed.
+//
+// Usage example:
+//
+// auto metrics_uploader = MetricsUploader::CreateMetricsUploader();
+// if (metrics_uploader.has_value()) {
+//   metrics_uploader.value()->SendLogEvent(...);
```

Comment:
Since `CreateMetricsUploader` now returns a value, it's either `metrics_uploader.value().SendLogEvent` or `metrics_uploader->SendLogEvent`.

Reply:
Thanks a lot! updated the comment